### ISSUE TITLE
Feature Request: Conform to Markdown Linters

### ISSUE THREAD

**Issue by D-Marc1** [jsdoc2md/jsdoc-to-markdown#213]
It would be really cool for this plugin's output to be able to conform to markdown linters, such as this one https://github.com/DavidAnson/markdownlint.

**Comment by 75lb** [comment]
jsdoc2md outputs Github-flavoured markdown.. does the linter above support GFM?

**Comment by D-Marc1** [comment]
Yes, but it enforces rules for consistency, like the max line length is 80 characters for instance. Much like how `eslint` works.

**Comment by D-Marc1** [comment]
Yes, I believe. I don't know all the rules fully by heart, so it's possible there might be a few rules it doesn't. But it seems to be working from me testing my own MD file.

**Comment by 75lb** [comment]
If you have [GFM](https://github.github.com/gfm/) issues, you could always try the `jsdoc2md --no-gfm` flag..

**Comment by 75lb** [comment]
I personally consider this low priority - it would be a considerable of work for not enough meaningful gain.. happy to accept PRs though.

**Comment by 75lb** [comment]
Re-opening.. it may take some time (given the complexity of the existing handlebars solution) but I will aspire to at least pass a GFM linting test..  I have made a start.

I'm not sure how many of the existing linting errors, for example those that complain about embedded HTML, can be fixed but I will take a look.. When this tool was created in 2014, things like HTML links were necessary to support both Github and the NPM website but things may have changed since.
